spec/fixtures/prover_outputs/CounterSatisfiable/Vampire

```diff
@@ -0,0 +1,29 @@
+Calculus      : VampireResCalc
+Spent time    : CPU time        = 00:00:00.295999999999
+                Wall clock time = 00:00:00.294999999999
+Prover output :
+Satisfiability detected
```

Comment:
Vampire says "Satisfiability detected" - but why is this then located in folder CounterSatisfiable?